extensions/Worker.Extensions.Http.AspNetCore/src/HostBuilderExtensions.cs

```diff
@@ -22,7 +24,7 @@ public static class HostBuilderExtensions
         public static IHostBuilder ConfigureAspNetCoreIntegration(this IHostBuilder builder)
         {
             // TODO: Update this logic. Port should always come through configuration.
-            var port = Environment.GetEnvironmentVariable("Azure_Functions_HttpProxyingPort") ?? "5555";
+            var port = Environment.GetEnvironmentVariable("AZURE_FUNCTIONS_HTTP_PROXY_PORT") ?? "5555";
```

Comment:
this logic will be updated later (port should always come through config) but @brettsam is this name better?

Reply:
Related change in host feature branch in [this PR](https://github.com/Azure/azure-functions-host/pull/9149/files).

Reply:
We should start with `FUNCTIONS_` -- I didn't realize that we have a bunch of env vars that use that prefix already, so let's stick with it.